<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditBrush</class>
 <widget class="QWidget" name="EditBrush">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>201</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditBrush</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#EditBrush {
	background-color: rgb(50, 50, 50);
}
* {
	font-family: &quot;Arial&quot;;
	font-weight: bold;
}
QLabel, QCheckBox {
	color: rgb(225, 225, 225);
	font: 500 8pt &quot;Arial&quot;;
}
QPushButton, QLineEdit, QSpinBox, QDoubleSpinBox {
	border: 1px solid rgb(75, 75, 75);
	background-color: rgb(35, 35, 35);
	color: rgb(200, 200, 200);
}
QPushButton:hover, QLineEdit:focus, QSpinBox:focus, QDoubleSpinBox:focus {
	border-color: rgb(0, 128, 255);
	color: rgb(255, 255, 255);
}
QPushButton:hover:pressed {
	background-color: rgb(25, 25, 25);
}

QSlider:disabled {
  background: #252525;
}

QSlider:focus {
  border: none;
}

QSlider::groove:horizontal {
  background: #505050;
  border: none;
  height: 2px;
  margin: 0px;
  border-radius: 4px;
}

QSlider::groove:vertical {
  background: #505050;
  border: none;
  width: 2px;
  margin: 0px;
  border-radius: 4px;
}

QSlider::add-page:vertical {
  background: #808080;
  border: 1px solid #32414B;
  width: 4px;
  margin: 0px;
  border-radius: 4px;
}

QSlider::add-page:vertical :disabled {
  background: #14506E;
}

QSlider::sub-page:horizontal {
  background: #808080;
  height: 4px;
  margin: 0px;
}

QSlider::sub-page:horizontal:disabled {
  background: #14506E;
}

QSlider::handle:horizontal {
  background: #323232;
  border: 2px solid #909090;
  width: 11px;
  height: 9px;
  margin: -7px 0px;
  border-radius: 7px;
}

QSlider::handle:horizontal:hover {
  border: 2px solid #cccccc;
}

QSlider::handle:vertical {
  background: #323232;
  border: 2px solid #909090;
  width: 9px;
  height: 11px;
  margin: 0px -7px;
  border-radius: 7px;
}

QSlider::handle:vertical:hover {
  border: 2px solid #cccccc;
}

QScrollBar:horizontal {
  height: 16px;
  margin: 2px 0px 2px 0px;
  background: #282828;
  border: 1px solid #323232;
}

QScrollBar:vertical {
  background: #282828;
  border: 1px solid #323232;
  width: 16px;
  margin: 0px 2px 0px 2px;
}

QScrollBar::handle:horizontal {
  margin: 1px;
  background-color: #404040;
  min-width: 8px;
  border-radius: 4px;
}

QScrollBar::handle:vertical {
  margin: 1px;
  background-color: #404040;
  min-height: 8px;
  border-radius: 4px;
}

QScrollBar::add-line:horizontal {
  border-image: url(&quot;:/qss_icons/rc/right_arrow_disabled.png&quot;);
  height: 10px;
  width: 10px;
  subcontrol-position: right;
  subcontrol-origin: margin;
}

QScrollBar::add-line:horizontal:hover, QScrollBar::add-line:horizontal:on {
  border-image: url(&quot;:/qss_icons/rc/right_arrow.png&quot;);
  height: 10px;
  width: 10px;
  subcontrol-position: right;
  subcontrol-origin: margin;
}

QScrollBar::add-line:vertical {
  border-image: url(&quot;:/qss_icons/rc/down_arrow_disabled.png&quot;);
  height: 10px;
  width: 10px;
  subcontrol-position: bottom;
  subcontrol-origin: margin;
}

QScrollBar::add-line:vertical:hover, QScrollBar::add-line:vertical:on {
  border-image: url(&quot;:/qss_icons/rc/down_arrow.png&quot;);
  height: 10px;
  width: 10px;
  subcontrol-position: bottom;
  subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal {
  border-image: url(&quot;:/qss_icons/rc/left_arrow_disabled.png&quot;);
  height: 10px;
  width: 10px;
  subcontrol-position: left;
  subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal:hover, QScrollBar::sub-line:horizontal:on {
  border-image: url(&quot;:/qss_icons/rc/left_arrow.png&quot;);
  height: 10px;
  width: 10px;
  subcontrol-position: left;
  subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical {
  border-image: url(&quot;:/qss_icons/rc/up_arrow_disabled.png&quot;);
  height: 10px;
  width: 10px;
  subcontrol-position: top;
  subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical:hover, QScrollBar::sub-line:vertical:on {
 border-image: url(&quot;:/qss_icons/rc/up_arrow.png&quot;);
  height: 10px;
  width: 10px;
  subcontrol-position: top;
  subcontrol-origin: margin;
}

QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal {
  background: none;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
  background: none;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
  background: none;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
  background: none;
}</string>
  </property>
  <widget class="QSlider" name="slide_min">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spin_min">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="prefix">
    <string>Min: </string>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spin_max">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="prefix">
    <string>Max: </string>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QSlider" name="slide_max">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_hard">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="prefix">
    <string>Hardness: </string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSlider" name="slide_hard">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
